demos/can-component/selected.html

```diff
@@ -24,7 +24,7 @@
 var DefineList = require("can-define/list/list");
 var stache = require("can-stache");
 
-var ViewModel = DefineMap.extend({
+var SelectViewModel = DefineMap.extend("SelectViewModel", {
```

Comment:
Constructor functions are usually capitalized, which is why I changed it from `selectVM` to `SelectViewModel`.

Also, we don’t show this everywhere, but if you provide a string as the first argument to `.extend()`, it’ll have a nicer name in the debugger. https://canjs.com/doc/can-define/map/map.extend.html